DribbbleSDKExample/DribbbleSDKExample/ApiCallFactory.m

```diff
@@ -26,7 +27,7 @@ + (ApiCallWrapper *)apiCallWrapperWithTitle:(NSString *)title selector:(SEL)sele
 }
 
 + (NSArray *)demoApiCallWrappers {
-    
+    AppDelegate *delegate = [AppDelegate delegate];
```

Comment:
remove data properties from delegate at all (and remove them from VC too). just make parametrized contructor for demoApiCallsWrappers